src/Sankey.js

```diff
@@ -0,0 +1,201 @@
+        .d(path)
+        .select(
+          elem("g.d3plus-Links", {
+            parent: this._select,
+            enter: {transform},
+            update: {transform}
+          }).node()
+        )
+        .config(assign(this._tooltip, false))
+        .config(
+          configPrep.bind(this)(
+            assign(this._shapeConfig, {
+              Path: {
+                strokeWidth: d =>
+                  Math.max(
+                    1,
+                    Math.abs(d.source.y1 - d.source.y0) *
+                      (d.value / d.source.value) -
+                      20
+                  )
+              }
+            }),
+            "shape",
+            "Path"
+          )
+        )
+        .render()
+    );
+
+    this._shapes.push(
+      new Rect()
+        .config(this._shapeConfig)
+        .config(this._shapeConfig.Path)
+        .data(nodes)
+        .height(d => d.y1 - d.y0)
+        .width(d => d.x1 - d.x0)
+        .x(d => (d.x1 + d.x0) / 2)
+        .y(d => (d.y1 + d.y0) / 2)
+        .select(
+          elem("g.d3plus-sankey-nodes", {
+            parent: this._select,
+            enter: {transform},
+            update: {transform}
+          }).node()
+        )
+        .config(configPrep.bind(this)(this._shapeConfig, "shape", "Rect"))
+        .render()
+    );
+    console.log(this);
+    return this;
+  }
+
+  nodeId(_) {
+    return arguments.length ? (this._nodeId = _, this) : this._nodeId;
+  }
+
+  /**
+      @memberof Network
+      @desc A predefined *Array* of edges that connect each object passed to the [node](#Network.node) method. The `source` and `target` keys in each link need to map to the nodes in one of three ways:
+1. The index of the node in the nodes array (as in [this](#) example).
+2. The actual node *Object* itself.
+3. A *String* value matching the `id` of the node.
+
+The value passed should either be an *Array* of data or a *String* representing a filepath or URL to be loaded. An optional formatting function can be passed as a second argument to this method. This custom function will be passed the data that has been loaded, as long as there are no errors. This function should return the final links *Array*.
+      @param {Array|String} *links* = []
+      @chainable
+  */
+  links(_) {
+    return arguments.length ? (this._links = _, this) : this._links;
+  }
+
+  /**
+      @memberof Sankey
+      @desc The list of nodes to be used for drawing the network. The value passed should either be an *Array* of data or a *String* representing a filepath or URL to be loaded.
+
+Additionally, a custom formatting function can be passed as a second argument to this method. This custom function will be passed the data that has been loaded, as long as there are no errors. This function should return the final node *Array*.
+      @param {Array|String} *nodes* = []
+      @chainable
+  */
+  nodes(_) {
+    return arguments.length ? (this._nodes = _, this) : this._nodes;
+  }
```

Comment:
The `@desc` for these `nodes` and `links` methods say they work with URL filepaths, but they don't :-)

Take a look at how it's done in Network, and port that logic over to here: https://github.com/d3plus/d3plus-network/blob/master/src/Network.js#L433-L442